src/app/features/cases/components/organisms/CaseTimeline/TimelineContainer.tsx

```diff
@@ -32,83 +31,43 @@ const Div = styled.div`
 }
 `
 
-// const StyledButton = styled(Button)`
-//   background-color: transparent;
-// `
-const data = [
-  { 
-    "id": 0,
-    "event_values": {},
-    "date_created": "2020-11-02T16:27:27.120Z",
-    "type": "DEBRIEFING",
-    "emitter_id": 0,
-    "case": 0
-  },{ 
-    "id": 1,
-    "event_values": {},
-    "date_created": "2020-11-01T16:27:27.120Z",
-    "type": "VISIT",
-    "emitter_id": 1,
-    "case": 1
-  },{ 
-    "id": 2,
-    "event_values": {},
-    "date_created": "2020-10-31T16:27:27.120Z",
-    "type": "VISIT",
-    "emitter_id": 2,
-    "case": 2
-  }
-]
+const StyledButton = styled(Button)`
+  background-color: transparent;
+`
 
 const TimelineContainer: React.FC<Props> = ({ caseId }) => {
-  //const { data } = useCaseEvents(caseId!)
-  
+  const data = useCaseEvents(caseId!).data
 
-  console.log("data", data)
-
-  const groupedEvents = data.reduce(function (r, a) {
-      r[a.type] = r[a.type] || []
-      r[a.type].push(a)
-      return r
-  }, Object.create(null))
-
-  const keys = Object.keys(groupedEvents)
+  const debriefEvents = data?.filter(event  => event.type === "DEBRIEFING")
```

Comment:
Cosmetic: you can also use destructuring within the paramater list. E.g. `const debriefEvents = data?.filter(({ type })  => type === "DEBRIEFING")`